presto-main/src/main/java/com/facebook/presto/operator/TaskStats.java

```diff
@@ -137,6 +139,7 @@ public TaskStats(
             @JsonProperty("userMemoryReservation") DataSize userMemoryReservation,
             @JsonProperty("revocableMemoryReservation") DataSize revocableMemoryReservation,
             @JsonProperty("systemMemoryReservation") DataSize systemMemoryReservation,
+            @JsonProperty("peakTotalMemoryInBytes") long peakTotalMemoryInBytes,
```

Comment:
Looking at cumulativeUserMemory, I'd expect

`@JsonProperty("peakTotalMemory") double peakTotalMemory,`

Reply:
Or `peakTotalMemoryReservation` and use `DataSize` as type? No strong opinion here.